src/Utils/FileSystem.php

```diff
@@ -131,11 +131,14 @@ public static function read(string $file): string
 
 	/**
 	 * Writes a string to a file.
+	 * @param $mode int|NULL
 	 * @return void
 	 * @throws Nette\IOException
 	 */
-	public static function write(string $file, string $content, int $mode = 0666)
+	public static function write(string $file, string $content, $mode = 0666)
 	{
+		assert($mode === NULL || is_int($mode));
+
```

Comment:
I usually use assert when there is no possibility of declaring type using typehint (union types, ...) Do you like it or should it fail at line [144](https://github.com/nette/utils/pull/139/files#diff-f0409c8a1df5e9194764902390ef6f9bR144)